src/images/DisplayBgImage.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+import { fetchAPI } from '../utils/fetchData';
+
+interface PictureOfDay {
+    date: string,
+    explanation: string,
+    hdurl: string,
+    title: string,
+    url: string
+}
+
+const DisplayBgImage = () => {
```

Comment:
Similar to the comment above please can this be renamed as `DisplayBackgroundImage` (please also rename the filename to match)